baseplate/sidecars/live_data_watcher.py

```diff
@@ -85,7 +85,13 @@ def get_data_to_write(json_data: dict, data: Optional[bytes]) -> Optional[bytes]
         # }
         # If the load type is 'S3', this format is an indication that we support
         # downloading the contents of encrypted files uploaded to S3.
-        if json_data.get("live_data_watcher_load_type") == "S3":
+        is_s3 = False
+        try:
+            is_s3 = json_data["live_data_watcher_load_type"] == "S3"
+        except (KeyError, TypeError):
+            is_s3 = False
+
+        if is_s3:
```

Comment:
Maybe it's just me but this feels like a very long way of doing something like:
```
if isinstance(json_data, dict) and json_data.get("live_data_watcher_load_type", None) == "S3":
    if not S3_FETCHER_ENABLED:
```

If we keep the try/except, your except statement doesn't do anything, that value is already false. So I would just `pass` at this point I think.